crypto/kyber/p_kyber512_asn1.c

```diff
@@ -16,7 +16,7 @@ static void kyber512_free(EVP_PKEY *pkey) {
 }
 
 static int kyber512_set_priv_raw(EVP_PKEY *pkey, const uint8_t *in, size_t len) {
-  if (len != KYBER512_PRIVATE_KEY_BYTES) {
+  if (len != KYBER512_SECRET_KEY_BYTES) {
```

Comment:
Can we statically assert there is space enough in `key->priv` in line 30? (https://github.com/awslabs/aws-lc/pull/605/files#diff-283a395b51f7e614369b99e205f13b02151037f16d48f4e5ca43733c93fd3554R30)

Reply:
Same for the public field in `kyber512_set_pub_raw()`.

Reply:
done